CHANGELOG.md

```diff
@@ -1,6 +1,41 @@
 Changes
 =======
 
+# 5.15.0 / 07-10-2017
+**Linux, Windows, Docker and Source Install**
+
+### Details
+https://github.com/DataDog/dd-agent/compare/5.14.1...5.15.0
+
+### Changes
+Please refer to the [Integrations Core Checks 5.15.0 tag](https://github.com/DataDog/integrations-core/releases/tag/5.15.0) for the list of changes on the core checks.
+Please refer to the [Trace Agent 5.15.0 tag](https://github.com/DataDog/datadog-trace-agent/releases/tag/5.15.0) for the list of changes on the Trace Agent.
+
+* [FEATURE] [kube] Allow to run agent on the node [#3401][]
+* [FEATURE] [core] Adds fraction of file handles in use over system limit [#3395][]
+* [FEATURE] [PrometheusCheck] add ignore_metrics array [#3394][]
+* [FEATURE] pkg: Add process-agent to supervisor and flares [#3409][]
+* [FEATURE] Get real host aliases for Cloud Foundry [#3406][]
+* [FEATURE] [docker] add docker/mesos version host tags [#3383][]
+* [FEATURE] [Windows] add additional calculator types [#3378][]
+* [FEATURE] Add k8s 1.6 to cgroup parsing dockerutil test [#3377][]
+* [FEATURE] Mesos tagging and generic container tagger [#3375][]
+* [FEATURE] [kubernetes] add kubernetes container name [#3368][]
+* [FEATURE] Add support for etcd auth [#3357][]
+* [FEATURE] Add support for %%id%% and %%pid%% template variables [#3404][]
+* [IMPROVEMENT] Add container name tag in auto discovery [#3421][]
+* [IMPROVEMENT] Mesos: extract chronos_job and chronos_job_owner if not empty [#3417][]
+* [IMPROVEMENT] Docker: disable mesos_task and chronos_job tags because of high cardinality [#3414][]
+* [IMPROVEMENT] Disable kube_service tagging on 4 403 errors from apiserver [#3413][]
+* [IMPROVEMENT] constant for container name label [#3405][]
```

Comment:
We always thank community contributions. So for example this one should also say: `thanks @_foo`